### DISCUSSION TITLE
How to use Pagination to have "Back" and "Next" in the buttons like this?

### DISCUSSION THREAD

**Discussion by hansyulian** [mantinedev/mantine#5852] [Q&A]
Hi, i would like to know on how to be able to add label to back and next like this:
![image](https://github.com/mantinedev/mantine/assets/10959546/3e1453a3-8744-4d4c-890a-f47a38a02ee2)
Thank you.

**Comment by tombohub** [comment]
Did you try `<Pagination.Next component={NextButton} href="#page-2" />` ?

**Reply by hansyulian** [reply]
is it like this?

```
  
function PreviousButton() {
  return <Group gap='xs'>
    <Image src={prevIcon} height={16} />
    <Text>Back</Text>
  </Group>;
}

 <Pagination size='sm' radius='xl' withControls={false} {...props} >
      <Pagination.Previous component={PreviousButton} />
      <Pagination.Next Component={NextButton}/>
 </Pagination>
```

this one doesn't work

**Reply by tombohub** [reply]
What's the error? Did you try like this: `<Pagination.Next Component={<NextButton/>} />`

**Reply by hansyulian** [reply]
for your way, it's message is ` Type 'Element' is not assignable to type 'ElementType<any, keyof IntrinsicElements>'` and it just doesn't render

**Reply by tombohub** [reply]
It works for me, paste the whole code
